qa/suites/rados/dashboard/tasks/dashboard.yaml

```diff
@@ -5,7 +5,7 @@ tasks:
       # tests may leave mgrs broken, so don't try and call into them
       # to invoke e.g. pg dump during teardown.
       wait-for-scrub: false
-      log-whitelist:
+      log-ignorelist:
```

Comment:
Not sure if this has already been discussed and settled, but I still find that `ignorelist-blocklist` lacks the clarity of `allowlist-denylist` (e.g.: as in https://github.com/kubernetes/kubernetes/issues/90277).